api/src/main/scala/dcos/metronome/api/v1/models/package.scala

```diff
@@ -248,4 +249,39 @@ package object models {
   }
 
   implicit lazy val MetronomeInfoWrites: Writes[MetronomeInfo] = Json.writes[MetronomeInfo]
+
+  implicit lazy val MarathonRunSpecWrites: Writes[RunSpec] = new Writes[RunSpec] {
+    override def writes(runSpec: RunSpec): JsValue = Json.obj(
+      "cpu" -> runSpec.cpus,
+      "mem" -> runSpec.mem,
+      "cmd" -> runSpec.cmd,
+      "container" -> runSpec.container.toString,
+      "args" -> runSpec.args,
+      "env" -> runSpec.env.toString(),
+      "constraints" -> runSpec.constraints.toString(),
+      "roles" -> runSpec.acceptedResourceRoles
+    )
+  }
+
+  implicit lazy val QueuedTaskInfoWrites: Writes[QueuedTaskInfo] = new Writes[QueuedTaskInfo] {
+    override def writes(taskInfo: QueuedTaskInfo): JsValue = Json.obj(
+      "runid" -> taskInfo.runSpec.id.toString,
```

Comment:
runId

Reply:
yeah... we translate so much from mesos terminology, marathon terminology to metronome.   I wish we all stuck more closely to mesos.   This would be a taskId in that case.   In this case, with metronome, there is a jobid, which is the id of the job.  and there are runids which is the id of each task or run of that job.